src/bootstrap/build/sanity.rs

```diff
@@ -78,6 +79,24 @@ pub fn check(build: &mut Build) {
             panic!("the iOS target is only supported on OSX");
         }
 
+        if cfg!(windows) {
```

Comment:
Could you add a comment to this explaining that this is just ensuring that we can run binaries like `llvm-config` and the compiler by ensuring that the dynamically linked libraries are in `PATH`?